qdax/baselines/sac.py

```diff
@@ -428,14 +542,18 @@ def _update_actor(
             New params and optimizer state. Current loss. New random key.
         """
         random_key, subkey = jax.random.split(random_key)
-        policy_loss, policy_gradient = jax.value_and_grad(self._policy_loss_fn)(
+        policy_loss, policy_gradient = jax.value_and_grad(sac_policy_loss_fn)(
             training_state.policy_params,
-            training_state.critic_params,
-            alpha,
+            policy_fn=self._policy.apply,
+            critic_fn=self._critic.apply,
+            parametric_action_distribution=self._parametric_action_distribution,
+            critic_params=training_state.critic_params,
+            alpha=jnp.exp(training_state.alpha_params),
             transitions=transitions,
-            random_key=subkey,
+            random_key=random_key,
```

Comment:
Same remark for the random key management.

Reply:
done